tools/walletextension/metrics/metrics.go

```diff
@@ -0,0 +1,83 @@
+package metrics
+
+import (
+	"sync"
+	"sync/atomic"
+	"time"
+)
+
+type MetricsTracker struct {
```

Comment:
the problem with this is that it's volatile. Any restart of the gateway will wipe the numbers